larq/quantizers.py

```diff
@@ -179,6 +206,7 @@ def __init__(self, clip_value: float = 1.0, **kwargs):
         super().__init__(**kwargs)
 
     def call(self, inputs):
+        super().call(inputs)
```

Comment:
Sorry, I'm not sure what you mean by the second option. Why is it better to do `inputs = super().call(inputs)` instead of just `super().call(inputs)`?

Reply:
I think just in general `super().call()` is expected to return a tensor again, so if someone would subclass the `BaseQuantizer` I would expect the following to work, right?
```python
def call(self, inputs):
    print("What shape is this?", inputs.shape)
    return super().call(inputs)
```

Reply:
At leas that's what [base layers do](https://github.com/tensorflow/tensorflow/blob/f6c196f73a5478dd8f7a428370770004a44b033b/tensorflow/python/keras/engine/base_layer.py#L282-L292)

Reply:
Ahh, I get it now, thanks. I've updated it in e6b636a.